### PR TITLE
feat(apps): add new service icons, enable apps feature

### LINKED ISSUE #4213: Need UX review for Connected Apps
@ryanfeeley @davismtl please visit: 

https://latest.dev.lcip.org/settings/clients?forceDeviceList=1&forceAppsList=1
http://accounts.firefox.com/settings?forceDeviceList=1&forceAppsList=1

and provide any feedback to unblock this feature and clear it to be enabled to users without a url flag.

On latest you can use these apps to test:

https://123done-content.dev.lcip.org/
https://321done-content.dev.lcip.org/

On prod you can use add-ons, pocket, etc.

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #4213 

@vbudhram please r?

### CODE REVIEW

File: app/tests/spec/views/settings/clients.js [hunk 1]
```diff
@@ -142,9 +142,6 @@ define(function (require, exports, module) {
       });
 
       it('renders attachedClients and apps if apps view enabled', function () {
```
Comment (vbudhram) [inline app/tests/spec/views/settings/clients.js]: Can probably remove `if apps view is enabled` since it is always true.

Comment (vbudhram) [inline app/tests/spec/views/settings/clients.js]: Similar to https://github.com/mozilla/fxa-content-server/blob/f8c9b1061028bc2ec8e3d2b72f696c6e5ec54552/app/tests/spec/views/settings/clients.js#L149, I think you can add a `properly sets the type of the app` that tests name and that correct place holder image is set for Pocket, Add-ons and other apps.

Comment (vladikoff) [inline app/tests/spec/views/settings/clients.js]: thanks!

**Comment by vladikoff** [comment]
Unrelated test failure on Circle..

**Pull request merged**